Packages/ohif-measurement-table/client/utils/retrieveDataFromMeasurements.js

```diff
@@ -0,0 +1,273 @@
+import { dcmjs } from 'meteor/ohif:cornerstone';
+
+const sopClassUid = dcmjs.data.DicomMetaDictionary.sopClassUIDsByName["CTImage"];
+
+const measurementGroupContentItem = (measurement) => {
```

Comment:
We should probably make it clear that this only handles length measurements.

Reply:
The CTImage shouldn't be used - instead the sopClassUID should be extracted from the dataset that is being measured.